### DISCUSSION TITLE
Will Lightspark run Adobe Captivate exe files?

### DISCUSSION THREAD

**Discussion by BenCrawford** [lightspark/lightspark#622] [Q&A] [answered]
Will this product Install via the windows 64 bit exe installer and make the library visible for Adobe Captivate exe files to run? Adobe Captivate exe files still rely on flash and can still be published as exe files but requires flash to publish. We have a collection of training courses that are going away in the next month or 2 but still need to be usable as of right now. I installed this application and it seems that either A the library needed isnt there or B it isnt installing correctly where it may need these files to be available. Any help would be appreciated.

**Comment by dbluelle** [comment, marked as answer]
I don't know anything about Adobe Captivate, but I assume it's some kind of wrapper around an swf file(?). So you need to have some tool to extract the swf file from the exe (which is not in the scope of lightspark). 
So unless such a tool exists, I don't think that lightspark is able to "play" such an exe.
